iidm/iidm-xml-converter/src/main/resources/xsd/iidm_V1_2.xsd

```diff
@@ -260,21 +261,43 @@
             </xs:extension>
         </xs:complexContent>
     </xs:complexType>
-    <xs:complexType name="ShuntCompensator">
+    <xs:complexType name="AbstractShuntCompensator">
```

Comment:
As this class doesn't exist in Java, we should remove it from the XSD. What do you think?

Reply:
I can simply call it `ShuntCompensator`? However, the xsd is not equivalent to Java structure (the Java classes `NonLinearShuntCompensator` and `LinearShuntCompensator` do not exist for example).

Reply:
ShuntCompensator seems more user friendly: keep in mind that the XSD is for the human.